features/apimgt/org.wso2.carbon.apimgt.core.feature/src/main/resources/sql/h2.sql

```diff
@@ -1420,9 +1420,10 @@ CREATE TABLE IF NOT EXISTS AM_APPLICATION (
     UPDATED_TIME TIMESTAMP,
     UUID VARCHAR(256),
     TOKEN_TYPE VARCHAR(10),
+    ORGANIZATION VARCHAR(100),
```

Comment:
Shall we make the size consistent? 100 vs 256

Reply:
Fixed in https://github.com/wso2/carbon-apimgt/pull/10585/commits/00b44d406db58adcab547df7bcb783088089fbb5